VSDiagnostics/VSDiagnostics/VSDiagnostics/Utilities/Extensions.cs

```diff
@@ -205,5 +206,62 @@ public static bool IsNameofInvocation(this InvocationExpressionSyntax invocation
 
             return identifier != null && identifier.Identifier.ValueText == "nameof";
         }
+
+        public static List<T> SyntaxNodeOfType<T>(this IEnumerable<SyntaxNode> enumerable, SyntaxKind kind) where T : SyntaxNode
```

Comment:
Let's stick with the same name `OfType` -- method overloading should not be distinguished based on name.